src/utils/apiConfiguration.ts

```diff
@@ -13,7 +13,7 @@ export function apiConfiguration(config?: ConfigurationParameters): Configuratio
         fetchApi: fetch as unknown as ConfigurationParameters['fetchApi'],
         headers: {
             ...config?.headers,
-            'Authorization': `Bearer ${config?.accessToken}`,
+            'Authorization': `Bearer ${config?.accessToken as string}`,
```

Comment:
Does the case of an undefined config need to be handled?

& a separate question, but why are `accessToken` and `headers.Authorization` both needed in this object?

Reply:
`apiConfiguration` is only ever called internally when constructing codegen options https://github.com/passageidentity/passage-flex-node/blob/7dae6eb25411d8d3c021c70cda68dbbad15a5a35/src/classes/PassageFlex/PassageFlex.ts#L31

and we take in our own values in `PassageFlexConfig` to populate this function's parameters https://github.com/passageidentity/passage-flex-node/blob/ceabe0373a02bc25da8a151d45ed395deb8c568b/src/classes/PassageFlex/types.ts#L4

where we always throw on undefined for the api key before this function is called https://github.com/passageidentity/passage-flex-node/blob/7dae6eb25411d8d3c021c70cda68dbbad15a5a35/src/classes/PassageFlex/PassageFlex.ts#L23-L24

so the value should never be undefined at this point. i changed the parameter to be required in 6c9991634979910148cdf62528094e81492f91b5

Reply:
> why are accessToken and headers.Authorization both needed in this object?

good point, i've removed the header in de63778b98752e5165925c597932c2356db324a0